app/src/main/res/menu/settings_menu.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:android="http://schemas.android.com/apk/res/android">
+    <item
+        android:id="@+id/menu_settings"
+        android:title="Settings"
+        app:showAsAction="always"
```

Comment:
This will have to be discussed later on, if we want to see those settings from every activity or only on the map !